src/Community/Contribute/index.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import CommunityButton from '../Button'
+import CommunityBlock from '../Block'
+import CommunitySection from '../Section'
+
+import data from '../data'
+
+import { Item, Items, Wrapper } from '../styles'
+
+const { description, title } = data.section.contribute
+
+export default function CommunityContribute({ theme }) {
+  return (
+    <Wrapper>
+      <CommunitySection
+        anchor="contribute"
+        background="/static/img/community/contribute_bg.jpg"
+        color={theme.color}
+        description={description}
+        icon="/static/img/community/contribute.svg"
+        title={title}
+      >
+        <Items>
+          <Item>
+            <CommunityBlock
+              title="Make a PR"
+              action={
+                <CommunityButton theme={theme} href="/">
+                  Open Chat
+                </CommunityButton>
+              }
+            >
+              Become DVC contributor and let us build something great together.
+            </CommunityBlock>
+          </Item>
+          <Item>
+            <CommunityBlock
+              title="Write a blogpost"
+              action={
+                <CommunityButton theme={theme} href="/">
+                  Learn More
+                </CommunityButton>
+              }
+            >
+              Have something cool on your mind? Suggest a text and we&apos;ll
+              publish it in our blog.
```

Comment:
We're always interested in guest writers for our blog. If you've got something to share, please reach out.

Reply:
Same...